### DISCUSSION TITLE
Custom Build Issues/quirks

### DISCUSSION THREAD

**Discussion by cburfordjr** [mriscoc/Ender3V2S1#662] [Q&A] [answered]
Great firmware. Running it on all 5 of my printers. I recently did some big modifications to one that required a custom compile of the firmware. This was simple enough. Followed the edit in Code and Marlin autobuild procedure and it built with no errors. The only things that were changed were the Printer Name, build size and disabled UBL and enabled ABL. It's all documented well in there so was a simple edit. After Flashing this to my printer, everything works, Only a couple of issues I have found. I am missing the bottom toolbar icons for the quick menu shortcuts. It's still there and selectable, just can't see it. There are no names or icons. Also noticed that when I change things like the screen color and probe count, those settings will not save to EEPROM and revert after a restart. other settings like steps/mm will save though.

These are minor annoyances, but I would like to know if I am missing something in the compile or if there is something that I can change to fix this. It was compiled for a v2 with 4.2.7 board.

Thanks in advance
-C

**Comment by mriscoc** [comment, marked as answer]
Check this: https://github.com/mriscoc/Ender3V2S1/issues/249

**Reply by cburfordjr** [reply]
Perfect. Got to protect your work. I will test that out in the morning and report back.

Thanks

**Reply by cburfordjr** [reply]
That took care of everything. Thanks for your help and keep up the great work.
